testing/frr/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Maintainer: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=frr
+pkgver=5.0.1
+pkgrel=0
+pkgdesc="FRRouting is a fork of quagga"
+pkgusers="frr"
+pkggroups="frr"
+url="https://frrouting.org/"
+arch="x86_64"
+license="GPL-2.0-only and LGPL-2.1-only"
+depends="json-c c-ares iproute2 python2 py-ipaddr bash"
+makedepends="ncurses-dev net-snmp-dev gawk texinfo perl
+    acct autoconf automake bash bison bsd-compat-headers
+    c-ares-dev cryptsetup-libs device-mapper-libs flex
+    json-c-dev kmod lddtree libacl libblkid libburn libedit
+    libisoburn libisofs libltdl libtool libuuid linux-headers lzo m4 mkinitfs
+    mtools pcre perl python2-dev readline-dev
+    squashfs-tools texinfo xorriso py-sphinx"
```

Comment:
are all those packages really needed to build this? I doubt that those are really needed:
- acct
- cryptsetup-libs
- device-mapper-libs
- kmod
- lddtree
- libacl
- libblkid
- libburn
- libedit
- libisoburn
- libisofs
- libltdl
- libuuid
- lzo
- m4
- mkinitfs
- mtools
- pcre
- squashfs-tools
- texinfo
- xorriso

Many of those are runtime libraries. some are tools needed to build alpine iso release image and simply does not make sense as makedepends for a routing software package.

This makes me question if the other makedependencies are needed too. It seems like makedepends was constructed by whatever happened to be installed on the build machine.

perl is specified twice.

Reply:
Will fix perl, makedepends are cut&paste, I'll start fresh and only add in what's necessary, that should clean it up...